.gitignore

```diff
@@ -89,3 +89,6 @@ tmp
 
 # asdf (not a typo! ;) used to manage multiple versions of tools
 .tool-versions
+
+# Development container configurations
+.devcontainer
```

Comment:
nit - this is for VScode right? If so, it's a good idea to mention that in the comment.

Reply:
Not exactly. I mean I use them with VScode yes, but there are other tools (even a CLI). I guess "development containers" sounds really generic but that's what they call it :sweat_smile: 
https://containers.dev/

Reply:
Let's add the link to the comment than it's fine